pytket/docs/changelog.rst

```diff
@@ -6,13 +6,17 @@ Changelog
 
 Minor new features:
 
+* Phasepolyboxes can be constructed from a given circuit
```

Comment:
```suggestion
* Phase polynomial boxes can be constructed from a given circuit
```
Please give the method name.

Reply:
The way to do this is just to use the constructor do you mean that?

Reply:
Right, something like "new constructor for `PhasePolyBox` given a circuit" -- i.e. mentioning the name of the affected class explicitly.

Reply:
Done in [](https://github.com/cqc-melf)@cqc-melf
[upadte changelog](https://github.com/CQCL/tket/pull/315/commits/40daea806485ac99fb45f5ce794b21c4ff3ca726)
[40daea8](https://github.com/CQCL/tket/pull/315/commits/40daea806485ac99fb45f5ce794b21c4ff3ca726)